magicctapipe/scripts/lst1_magic/lst1_magic_mc_dl0_to_dl1.py

```diff
@@ -224,83 +223,24 @@ def mc_dl0_to_dl1(input_file, output_dir, config):
             tels_with_trigger = event.trigger.tels_with_trigger
 
             # Check if the event triggers both M1 and M2 or not
-            trigger_m1 = tel_id_m1 in tels_with_trigger
-            trigger_m2 = tel_id_m2 in tels_with_trigger
+            if((set(MAGICs_IDs).issubset(set(tels_with_trigger))) and (MAGICs_in_use=="MAGIC1_MAGIC2")):
```

Comment:
I guess you could make it in one line 
magic_stereo = ... and ....

Reply:
done